README.md

```diff
@@ -124,6 +65,55 @@ You can build the initramfs built by u-root into the kernel via the
 `CONFIG_INITRAMFS_SOURCE` config variable or you can load it separately via an
 option in for example Grub or the QEMU command line or coreboot config variable.
 
+## Init and Uinit
+
+u-root has a very simple (exchangable) init system controlled by the `-initcmd`
+and `-uinitcmd` command-line flags.
+
+*   `-initcmd` determines what `/init` is symlinked to.
+*   `-uinitcmd` is run by the default u-root [init](cmds/core/init) after some
+    basic file system setup. There is no default, users should supply their own.
+*   If [init](cmds/core/init) finds no uinit to run, it will start a shell
+    determined by the `-defaultsh` argument.
+
+We expect most users to keep their `-initcmd` as [init](cmds/core/init), but to
+supply their own uinit for additional initialization or to immediately load
+another operating system.
+
+All three command-line args accept both a u-root command name or a target
+symlink path. **Only `-uinitcmd` accepts command-line arguments, however.**. For
+example:
+
+```bash
+$ u-root -uinitcmd="echo Go Gopher" ./cmds/core/{init,elvish}
+2020/04/30 21:05:57 could not create symlink from "bin/uinit" to "echo": command or path "echo" not included in u-root build: specify -uinitcmd="" to ignore this error and build without a uinit
+
+$ u-root -uinitcmd="echo Go Gopher" ./cmds/core/{init,echo,elvish}
+```
+
+boots as:
+
+```console
+2020/05/01 04:04:39 Welcome to u-root!
+                              _
+   _   _      _ __ ___   ___ | |_
+  | | | |____| '__/ _ \ / _ \| __|
+  | |_| |____| | | (_) | (_) | |_
+   \__,_|    |_|  \___/ \___/ \__|
+
+Go Gopher
+~/>
+```
+
+```bash
+// We don't presume to know whether your symlink target is correct or not.
```

Comment:
So it's actually bash:
```suggestion
# We don't presume to know whether your symlink target is correct or not.
```

Reply:
yeah, i moved this to be real text and not in the bash box